docs/coding-style.md

```diff
@@ -194,13 +228,46 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
 * Var pattern `o is { P: var p }` can be used only where `o` can be `null` and `p` is used at least 3 times.
 * Each compiler directive outside method body (namely `#if`/`#endif`) should be preceded and followed by an empty line.
 * Do not use `nullable`.
+* Avoid single-use variables, unless they really bring readability value.
+* Use file-scoped namespaces.
+* Tested variable is on the left, like `iterated == searchParameter` (where `iterated` is the tested variable)
+* If a string is multiline, use raw string literals, indented one tab-in from the declaration:
+```
+var raw = """
+    hello
+    my friend
+	""";
+```
+* Always use multi-line initializers for collection and objects, e.g.:
+```
+var thingy = new Thingy
+{
+	x = "hello",
+	y = 42,
+}
+
+var collection = new Dictionary<string, int>
+{
+	{ "hey" : 1 },
+	{ "there": 42 },
```

Comment:
Wrong sytnax.
```suggestion
	["hey"] = 1,
        ["there"] = 42
```

Reply:
It's my typo, sorry for that Andrei.
And Tim, the one I wanted to suggest is this one:

```csharp
        var dict = new Dictionary<int,int>
        {
            { 1, 2 },
            { 3, 4 },
        };
```